### ISSUE TITLE
Wrong acs module in 2.0.42 CLI -- AKS log analytics are broken

### ISSUE THREAD

**Issue by mboersma** [Azure/azure-cli#6933]
```console
$ /usr/local/bin/az --version
azure-cli (2.0.42)

acr (2.1.0)
acs (2.2.0)
```

The `acs` submodule should be `2.2.2` but didn't get updated. So #6782 isn't in there, and the documentation that describes creating an AKS cluster without an explicit log analytics workspace for monitoring is broken.

cc: @ganga1980 

**Environment summary**
Tested with the homebrew packaging as well as CloudShell.

**Comment by ganga1980** [comment]
@berndverst,  we fixed the exception in version 2.0.43. And also I see the acs module version as 2.2.2 in  Az CLI version 2.0.43 . Can you please upgrade to 2.0.43 and let us know still you see this issue?

az --version
azure-cli (2.0.43)
acr (2.1.2)
acs (2.2.2)

**Comment by mboersma** [comment]
@ganga1980 @berndverst sorry for getting my wires crossed. Version 2.0.43 got published to homebrew just after we opened this issue, and it is of course fixed:

```console
$ brew upgrade
==> Upgrading 1 outdated package, with result:
azure-cli 2.0.42 -> 2.0.43
$ /usr/local/bin/az --version | head
azure-cli (2.0.43)

acr (2.1.2)
acs (2.2.2)
```
